core/tests/swfs/avm1/object_function/output.txt

```diff
@@ -0,0 +1,9 @@
+[type Object]
```

Comment:
Can you please add `typeof` to these, and prefix them with what you're testing? The ones with `toString()` implemented will not show as broke if this method doesn't work, because `trace(11234)` is the same as `trace(Object(11234))`